packages/flutter/lib/src/rendering/sliver_multi_box_adaptor.dart

```diff
@@ -214,6 +219,31 @@ abstract class RenderSliverMultiBoxAdaptor extends RenderSliver
   /// The nodes being kept alive despite not being visible.
   final Map<int, RenderBox> _keepAliveBucket = <int, RenderBox>{};
 
+  List<RenderBox> _debugDanglingKeepAlives;
+
+  /// Indicates whether integrity check is enabled.
+  ///
+  /// Setting this property to true will immediately perform an integrity check.
+  ///
+  /// The integrity check includes:
+  ///
+  /// 1. Verify that the children index in childList is in ascending order.
+  /// 2. Verify that there is no dangling keepalive child as the result of [move].
+  bool get debugChildIntegrityEnabled => _debugChildIntegrityEnabled;
+  bool _debugChildIntegrityEnabled = true;
+  set debugChildIntegrityEnabled(bool enabled) {
+    assert(enabled != null);
+    assert(() {
+      _debugChildIntegrityEnabled = enabled;
+      _debugVerifyChildOrder();
+      if (_debugChildIntegrityEnabled) {
+        assert(!_debugDanglingKeepAlives.isNotEmpty);
+      }
+      return true;
+    }());
+
```

Comment:
nit: extraneous blank line

(general advice: it's often useful to self-review a patch to catch this kind of thing.)